src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -306,4 +306,10 @@ public net.minecraft.world.gen.layer.LayerUtil func_202829_a(JLnet/minecraft/wor
 private-f net.minecraft.world.server.ChunkHolder field_219320_o # block update location
 public net.minecraft.world.server.ServerChunkProvider field_186029_c # chunkGenerator
 public net.minecraft.world.server.ServerChunkProvider field_73251_h # worldObj
-public net.minecraft.world.storage.FolderName <init>(Ljava/lang/String;)V # constructor
\ No newline at end of file
+public net.minecraft.world.storage.FolderName <init>(Ljava/lang/String;)V # constructor
+protected-f net.minecraft.tags.NetworkTagManager field_215299_d # entityTypes
+protected-f net.minecraft.tags.NetworkTagManager field_205705_c # fluids
+protected-f net.minecraft.tags.NetworkTagManager field_199720_b # items
+protected-f net.minecraft.tags.NetworkTagManager field_199719_a # blocks
+public net.minecraft.world.storage.FolderName <init>(Ljava/lang/String;)V # constructor
+private-f net.minecraft.world.raid.Raid$WaveMember field_221284_f # VALUES
```

Comment:
You need to run checkSAS

Reply:
what for exactly? just ran and nothing changed

Reply:
You're including changes that you did not do.
https://github.com/MinecraftForge/MinecraftForge/blob/6bb0c33ef9d7917b975c2e4dcc50e7b38bb5c65e/src/main/resources/META-INF/accesstransformer.cfg#L272
See all those duplicate lines?
Also my bad, it's checkATs not checkSAS. It'll sort the ATs, make sure they are valid, remove duplicates, and propagate to children if needed.
Either way be sure to include only the changes you do.

Reply:
I think Lex meant checkAT

Reply:
ah, i see